src/scripts/anti-crise-scrapper.py

```diff
@@ -0,0 +1,112 @@
+import datetime
+from bs4 import BeautifulSoup
+from datetime import timedelta
+from src.models.Promotion import Promotion
+from src.scripts.utils import simple_get, promotion_post
+from src.scripts.scrapper import Scrapper
+
+USER_ID = 1
```

Comment:
What's this ?

Reply:
When we are creating a promotion it must be related to a user